R/palaeorotate.R

```diff
@@ -281,7 +318,8 @@ palaeorotate <- function(occdf, lng = "lng", lat = "lat", age = "age",
     }
     # Reconstruction files
     rot_files <- list(
-      BASE = "https://zenodo.org/record/7390066/files/",
+      BASE = "https://zenodo.org/record/7615203/files/",
```

Comment:
Is there a way to always link to the most recent version of the zenodo repo? (I know you can do this with the doi, but I don't know whether it's possible for files)

Reply:
I don't think so. Each version of the data has a unique DOI so you can identify which version of the data you are working on/with.

Reply:
Looks like `httr::GET("https://doi.org/10.5281/zenodo.7390065")$url` will always give you the most recent base URL for the files.

Reply:
`httr::GET("https://zenodo.org/record/7390065")$url` works too

Reply:
Implemented! Thanks!